src/core/Table/Table.tsx

```diff
@@ -638,6 +641,12 @@ export const Table = <
               'iui-zebra-striping': styleType === 'zebra-rows',
             }),
           })}
+          ref={bodyRef}
+          onScroll={() => {
+            if (headerRef.current && bodyRef.current) {
+              headerRef.current.scrollLeft = bodyRef.current.scrollLeft;
```

Comment:
Do you really need bodyRef if you have this inline listener? Might be able to get it from the event.

But I guess it's more clear this way